docs/components/movement-sensor/gps/set-up-base-station.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Set up a SparkFun RTK Reference Station"
+linkTitle: "SparkFun RTK Reference Station Setup"
+weight: 20
+type: "docs"
+description: "Follow this guide to set up your own SparkFun RTK base station, a high-precision GPS module designed for use with the RTK satellite navigation technique."
+images: ["/icons/components/imu.svg"]
```

Comment:
```suggestion
images: ["/icons/components/imu.svg"]
no_component: true
```

Reply:
and then I think you need to put 

```
toc_hide: true
```

in the frontmatter of the movement-sensor models. 

and remove `hide_children: true` from the mvoement-sensor/_index.md file

I think that's how the camera one is done

Reply:
oh and this should be moved out of the gps folder. GPS folder no longer exists...

Reply:
@npentrel Done, I think, it shows up the same as camera calibrate now.